test/test_transfer.cpp

```diff
@@ -78,7 +78,7 @@ struct test_storage : default_storage
 	{}
 
 	void set_file_priority(std::vector<std::uint8_t> const& p
-		, int const flags, storage_error& ec) override {}
+		, int flags, storage_error& ec) override {}
```

Comment:
I see. at some point it would probably make sense to silence that warning. it's just msvc saying that they used to be non-conforming.. but since master requires C++11, it seems reasonable to expect a conforming implementation (within reason, msvc-14.0 isn't all the way there yet)